packages/discord.js/typings/index.d.ts

```diff
@@ -2453,10 +2453,27 @@ export class Webhook extends WebhookMixin() {
   public sourceChannel: NewsChannel | APIPartialChannel | null;
   public token: string | null;
   public type: WebhookType;
-  public isIncoming(): this is this & { token: string };
+  public applicationId: Snowflake | null;
+  public isUserCreated(): this is this & {
+    type: WebhookType.Incoming;
+    applicationId: null;
+    owner: User | APIUser;
+    token: null;
+  };
+  public isApplicationCreated(): this is this & {
+    type: WebhookType.Incoming;
+    applicationId: Snowflake;
+    owner: null;
```

Comment:
This isn't correct, if there is an application id, then the `owner` is obviously defined!